integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWDTModelNoServer.java

```diff
@@ -0,0 +1,466 @@
+  private static final String MII_IMAGE_NAME = "wdtmodelnoserver-mii";
+  private static final int replicaCount = 2;
+  private static final String internalPort = "8001";
+  private static final String appPath = "sample-war/index.jsp";
+  private static final String adminServerPodName = domainUid + "-adminserver";
+  private static final String managedServerPrefix = domainUid + "-" + MANAGED_SERVER_NAME_BASE;
+  private static final String clusterName = "cluster-1";
+
+  private static String domainNamespace = null;
+  private static String opNamespace = null;
+  private static LoggingFacade logger = null;
+  private static String imageName = null;
+
+  /**
+   * Get namespaces for operator and WebLogic domain.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+
+    logger = getLogger();
+
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // get a unique domain namespace
+    logger.info("Getting a unique namespace for WebLogic domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+  }
+
+  /**
+   * Test in the WDT model file there is no AdminServerName and no Server section defined.
+   * Verify AdminServer Pod "domainUid-adminserver" is running and the defined WebLogic cluster is up and running.
+   */
+  @Test
+  @DisplayName("Test in the WDT model file there is no AdminServerName and no Server section defined")
+  void testWdtModelNoAdminServerNameNoServer() {
+
+    String wdtModelFile = "model-noadminservername-noserver.yaml";
+    imageName = createAndVerifyDomainImage(wdtModelFile);
+
+    // Verify the domain is created. The admin server pod "domainUid-adminserver" is up and running.
+    // Also verify the cluster is up and running.
+    createMiiDomainAndVerify(domainNamespace, domainUid,
+        imageName, adminServerPodName, managedServerPrefix, replicaCount);
+
+    //check and wait for the application to be accessible in all server pods
+    for (int j = 1; j <= replicaCount; j++) {
+      String managedServerPodName = managedServerPrefix + j;
+      String expectedStr = "Hello World, you have reached server " + MANAGED_SERVER_NAME_BASE + j;
+
+      logger.info("Checking that application is running on managed server pod {0}  in namespace {1} with "
+          + "expectedString {3}", managedServerPodName, domainNamespace, expectedStr);
+      checkAppIsRunning(
+          domainNamespace,
+          managedServerPodName,
+          expectedStr);
+    }
+
```

Comment:
Do you think it will be good to check the name of the admin server from domain runtime and see if it matches the expected admin server name?

Reply:
Same for all usecases

Reply:
Do you mean to check the MBean? I will add the check.

Reply:
added the check.